index.html

```diff
@@ -185,7 +180,7 @@ <h3 id="mission">Polyhack is one of the few hackathons that allows people to wor
 		<div class="row tealbg" id="Designers">
 			<div class="col-sm-7 col-sm-push-5 right">
 				<h1 class="right-title">Designers</h1>
-				<h3 id="mission">Polyhack is the only hackathon that runs a series of sponsored design competitions, also known as the <a href="http://go.tufts.io/design-wars">Design Wars</a>, as part of the event. You can work with a hackathon team and/or work on the design prompts all night long. After reviewing all the design submissions at the end of the day, the top 3 winners will get portfolio reviews with either Frog, Palantir, or Jacob Waites (who is an Interaction Designer at IDEO CoLAB).</h3>
+				<h3 id="mission">Polyhack is the only hackathon that runs a series of sponsored design competitions, also known as the <a href="go.tufts.io/mentor">Design Wars</a>, as part of the event. You can work with a hackathon team and/or work on the design prompts all night long. After reviewing all the design submissions at the end of the day, the top 3 winners will get Sketch licenses and portfolio reviews with either Frog, Palantir, or Jacob Waites (who is an Interaction Designer at IDEO CoLAB).</h3>
```

Comment:
you need http:// for the href to work. else it googles it or adds it to the end of the website name